frontend/views/Users.vue

```diff
@@ -65,7 +65,7 @@ export default {
   components: { Menu, Pagination },
   data() {
     return {
-      perPage: '',
+      perPage: this.$store.state.config.pagination[0],
```

Comment:
Missing undefined check for older configs / compatibility issue

Reply:
I changed the set config a bit to allow to set sensible defaults in the store. This reduces the need to use undefines an the messy code that comes with it.